apps/emqx/src/emqx_trace/emqx_trace_handler.erl

```diff
@@ -68,17 +70,29 @@
     Type :: clientid | topic | ip_address,
     Filter :: emqx_types:clientid() | emqx_types:topic() | string(),
     Level :: logger:level() | all,
-    LogFilePath :: string()
+    LogFilePath :: string(),
+    Formatter :: plain | json
```

Comment:
the main log config uses `text` and `json`. i think we should follow.
```suggestion
    Formatter :: text | json
```

Reply:
Fixed: [10957e7](https://github.com/emqx/emqx/pull/12863/commits/10957e7d79bbc62d26d8b9a41e8de26c7b3aa9f5)